dev-java/ant_core/ant_core-1.10.7.recipe

```diff
@@ -54,7 +54,7 @@ BUILD()
 {
 	source /system/data/profile.d/openjdk8.sh
 	export PATH=$JDK8_HOME/bin:$PATH
-	ANT_HOME= ./build.sh
+	ANT_HOME=./build.sh
```

Comment:
This line seems nonsensical. It basically makes the `BUILD` phase do nothing; the compilation then happens on the `INSTALL` phase. Was this a typo?

Reply:
With this change, build.sh isn't executed anymore.

Reply:
In the `BUILD` phase at least; what I want to know is why? This also isn't a particularly clean way to disabling it if that was intended.

Reply:
> With this change, build.sh isn't executed anymore.

Oh, that could explain why it worked, it was confusing for me as well. For some reason, the recipe was broken, due to JAVA_HOME not being defined. I will remove that line for the time being.

Reply:
Yes, but I'm fairly sure that we have a policy to keep building in BUILD. You could just define JAVA_HOME in BUILD as well.

Reply:
Hm, the problem is that the `./build.sh` line also seems to be the culprit for installing the program itself.

I will move the line there for the time being and make the appropriate changes later.

Reply:
Hm? I haven't looked much into it, but doesn't seem to install it if you don't pass install-lite and/or ANT_HOME is empty.